.storybook/stories/DataFiltering/TableFilteringMedium.tsx

```diff
@@ -30,15 +44,63 @@ export const TableFilteringMedium = ({
 }: {
 	selectable?: boolean;
 }) => {
+	const { updateData } = useDataContext();
+
+	const sectionAlertRef = useRef(null);
 	const [isOpen, toggleIsOpen] = useToggleState(false, true);
 
+	const [currentItem, setCurrentItem] = useState<BusinessForAudit>();
+	const [showSuccessMessage, setShowSuccessMessage] = useState(false);
+
+	const formRef = useRef(null);
+	const [isDrawerOpen, openDrawer, closeDrawer] = useTernaryState(false);
+	const onSubmitForm: MouseEventHandler<HTMLButtonElement> = (event) => {
+		event.preventDefault();
+
+		if (!currentItem) return;
+		updateData?.({
+			...currentItem,
+			...(radioAssigneeValue && { assignee: radioAssigneeValue }),
+		});
+
+		setShowSuccessMessage(true);
+
+		closeDrawer();
+	};
+	const onOpenDrawer = (newCurrentItem?: BusinessForAudit) => {
+		openDrawer();
+		setShowSuccessMessage(false);
+		setCurrentItem(newCurrentItem);
+		setRadioAssigneeValue(newCurrentItem?.assignee);
+	};
+
+	const [radioAssigneeValue, setRadioAssigneeValue] = useState<string>();
+	const handlerForKey = useCallback(
+		(key: string) => () => setRadioAssigneeValue(key),
+		[]
+	);
+	const isChecked = (key: string) => key === radioAssigneeValue;
+
 	// IDs for accordion to ensure accessibility
 	const buttonId = 'filter-button';
 	const bodyId = 'filter-body';
 
 	return (
 		<Stack gap={1.5}>
 			<H2 id={headingId}>Audits</H2>
+
+			<Box css={showSuccessMessage ? undefined : { display: 'none' }}>
+				<SectionAlert
```

Comment:
As discussed, needs to be dismissable

Reply:
Yeah, that was the other thing, thanks.

- [ ] section alert dismissable